src/tests/specs/common.ts

```diff
@@ -1,27 +1,55 @@
 import * as path from "path";
```

Comment:
This file was heavily rewritten, but it's now a mess. I focused on getting the tests out he door, the test infrastructure is bound to get refactored soon. I'll open an issue.